Dialog/Dialog.jsx

```diff
@@ -8,13 +8,29 @@ class Dialog extends MaterialComponent {
   constructor() {
     super();
     this.componentName = "dialog";
+    this._onAccept = this._onAccept.bind(this);
+    this._onCancel = this._onCancel.bind(this);
   }
   componentDidMount() {
     this.MDComponent = new MDCDialog(this.control);
+    this.MDComponent.listen("MDCDialog:accept", this._onAccept);
+    this.MDComponent.listen("MDCDialog:cancel", this._onCancel);
   }
   componentWillUnmount() {
+    this.MDComponent.unlisten("MDCDialog:accept", this._onAccept);
```

Comment:
The method will always exist, no? I mean we could check `this.MDComponent` instead, if that's necessary